test.sh

```
@@ -0,0 +1 @@
+          curl -I http://localhost:8080 && exit 0 || { echo "❌ Nginx test failed for ${{ matrix.repo }}"; exit 1; }
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)